shared/cfg/machines.cfg

```diff
@@ -17,6 +17,7 @@ variants:
         # port of pcie-switch, see RHBZ#1380285
         no WinXP Win2000 Win2003 RHEL.5 RHEL.6
         pci_bus = pci.0
+        pcie_extra_root_port = 3
```

Comment:
Hi,
I think it is inconvenient that need to declare the parameter "pcie_extra_root_port" to 0 again if boot up a guest with multi-disks. it seems like the "pcie_extra_root_port = 3" is a default value, the "pcie_extra_root_port = 0" is a custom value.This is not accord with logic. it should be fixed in case level.
Thanks.

Reply:
Well it's a tough call, there are many hotplug tests that used to work well out of the box on i440fx, but on q35 require extra root ports. Most tests don't require all pci slots, therefor occupying some by extra root ports is not an issue, which is why 3 became a default (note that IIRC libvirt does the same by default).

So changing to 0 is possible, but would require many fixes in individual tests.

Reply:
@ldoktor @YongxueHong I am checking if it is possible to insert the device to the extra root port when there is no sufficient slot, which might be a possible solution, and more like the real word case.

Reply:
well alternative solution would be to add root ports at the end of the vm creation and only add them if possible. That way one would get machine with 0 - $pcie_extra_root_port depending on the number of free root ports (or actually we could insert root ports to all free pci ports, but that might  cause other issues like boot problems on arm due to too many pci devices)